src/main/java/io/codelex/loops/exercises/RollTwoDice.java

```diff
@@ -0,0 +1,36 @@
+package io.codelex.loops.exercises;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class RollTwoDice {
```

Comment:
nedaudz formatings, biški pietrūkst spaces un par daudz tukšo līniju, ir arī citās vietas, bet to vairs tad nepieminēšu